corehq/apps/app_manager/static/app_manager/js/releases/releases.js

```diff
@@ -1,4 +1,22 @@
-hqDefine('app_manager/js/releases/releases', function () {
+hqDefine("app_manager/js/releases/releases", [
```

Comment:
The script tag for this file is [here](https://github.com/dimagi/commcare-hq/blob/a83cc4ab5816f17e0963d4c0b83efbe8349ecea3/corehq/apps/app_manager/templates/app_manager/app_view.html#L11).

`app_manager/js/download_async_modal` is directly above it.

`app_manager/js/menu` is in the parent template [here](https://github.com/dimagi/commcare-hq/blob/a83cc4ab5816f17e0963d4c0b83efbe8349ecea3/corehq/apps/app_manager/templates/app_manager/managed_app.html#L17).

The other dependencies are all widely-used HQ dependencies that are included earlier in `hqwebapp/base.html`